### PR TITLE
feat[WIP]: add Navbar component

### PR DESCRIPTION
**Pull request by saadahmadhere**
This PR includes Navbar and Links to Home and Login route.

### CODE REVIEW

File: src/Components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -1,7 +1,32 @@
...
+import { Link } from "react-router-dom";
 
 const Navbar = () => {
-  return <>Navbar</>;
+  return (
+    <>
+      <nav className="navbar flex">
+        <Link to="/">
+          <div className="logo">
+            <div className=" logo_text txt_semibold">stopphur</div>
```
Comment (shivampandey0) [inline src/Components/Navbar/Navbar.jsx]: Try to use the **Heading** tag here

**Review by shivampandey0 — APPROVED** [review]
Good code structure. Use of CSS variables.
Suggestion: try to use semantic HTML

**Review by abhij1607 — COMMENTED** [review]
Clean code | | All good practices followed

File: src/Components/Navbar/Navbar.jsx (hunk 1 above)
Comment (saadahmadhere) [reply src/Components/Navbar/Navbar.jsx]: ok, will reconsider that.

**Pull request merged**